content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Veneers"
+linkTitle: "Veneers"
+weight: 5
+date: 2022-06-30
+---
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions. FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/). However, FBCs can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`, a simpler transitional step to generating an FBC.
+
+## Design
+
+There are two components to every `veneer`:
+1. An arbitrary API
+2. An executable which processes #1 and produces a valid FBC.
+
+## Benefits
+
+- A set of consistent expectations from including as standard functionality with `opm`
+- A good set of use-cases for simplifying the ability of operator authors to manage FBCs, including contributing them to an index
+- A transitive benefit from consistency of practice as authors move from operator to operator
```

Comment:
Would we not put the info in a more friendly ( end-user way)?

Example: When should I use it?